jwst/lib/set_telescope_pointing.py

```diff
@@ -1241,14 +1241,14 @@ def calc_transforms_track_tr_202111(t_pars: TransformParameters):
     # Check on telemetry for FGS ID. If invalid, use either user-specified or default to 1.
     fgsid = t_pars.pointing.fgsid
     if fgsid not in FGSIDS:
-        logger.warning(f'Method {t_pars.method} requires a valid FGS ID in telementry.')
-        logger.warning('However telemetry reports an invalid id of {fgsid}')
+        logger.warning("Method %s requires a valid FGS ID in telementry.", t_pars.method)
+        logger.warning("However telemetry reports an invalid id of %s", fgsid)
```

Comment:
This was actually a bug before: f-string format without declaring it a f-string. Could not have been caught by style check. I just happened to notice it while fixing the line right above it.